### ISSUE TITLE
AssertionError

### ISSUE THREAD

**Issue by nanashi019** [FTM-Labs/Aptos-NFT-Mint#52]
![Screen Shot 2022-10-31 at 15 50 44](https://user-images.githubusercontent.com/101402057/198948530-17a5217e-011e-49f2-afc5-237a919fc03d.png)

anyone knows how to fix it??
I tried to create candymachine on devnet

**Comment by DloomPlz** [comment]
Having the same issue ! `Transaction Executed and Committed with Error LINKER_ERROR`

**Comment by shockValue666** [comment]
i have the same error

**Comment by kc-hsh-value** [comment]
i resolved it by deleting the repo and cloning it again in a different directory

**Comment by BlessedSiden1234** [comment]
Yeah it doesn't work on devnet, switch to mainnet or testnet and fund your account with at least 2 aptos it would work fine. Also remember to switch both the constant.py file and candymachineInfo.js files to mainnet or whichever network your switched to so it would work
